internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -1839,14 +1796,19 @@ func ExpandVirtualMachineScaleSetRollingUpgradePolicy(input []interface{}) *comp
 
 	raw := input[0].(map[string]interface{})
 
-	return &compute.RollingUpgradePolicy{
-		EnableCrossZoneUpgrade:              utils.Bool(raw["cross_zone_upgrades_enabled"].(bool)),
```

Comment:
moved below, this cannot be specified in default case, even set to false.
Message="EnableCrossZoneUpgrade property in Rolling Upgrade policy can not be set since this Virtual machine Scale Set is not zonal." Target="upgradePolicy.rollingUpgradePolicy.enableCrossZoneUpgrade"